qubrabench/algorithms/search.py

```diff
@@ -58,9 +58,11 @@ def search(
             N, T, error, max_classical_queries
         )
 
+
     # run the classical sampling-without-replacement algorithms
-    rng.shuffle(iterable)  # type: ignore
-    for x in iterable:
+    for _ in range(0, estimated_size):
+        sample = urandom(int(np.ceil(np.log2(float(estimated_size)))))
```

Comment:
is this using some arbitrary randomness? remember we should always only use `rng` to generate any randomness.